app/_landing_pages/insomnia.yaml

```diff
@@ -91,37 +81,32 @@ rows:
                 - title: Enterprise
                   key: ee
               features:
-                - title: Local vault for local storage of collections, design specs, and every other resource
+                - title: Local storage
                   oss: true
                   individual: true
                   team: true
                   ee: true
-                - title: Debug APIs using the most popular protocols and formats
+                - title: Access to API testing suite
```

Comment:
I'm wondering if this makes it sound like it's an existing test suite, and not a feature to create test suites?

Reply:
How about "API testing tools"